zh/deploy-on-gcp-gke.md

```diff
@@ -155,29 +157,69 @@ Terraform 自动加载和填充匹配 `terraform.tfvars` 或 `*.auto.tfvars` 文
 
     Outputs:
 
-    how_to_connect_to_default_cluster_tidb_from_bastion = mysql -h 172.31.252.20 -P 4000 -u root
     how_to_ssh_to_bastion = gcloud compute ssh tidb-cluster-bastion --zone us-west1-b
-    how_to_set_reclaim_policy_of_pv_for_default_tidb_cluster_to_delete = kubectl --kubeconfig /.../credentials/kubeconfig_tidb-cluster get pvc -n tidb-cluster -o jsonpath='{.items[*].spec.volumeName}'|fmt -1 | xargs -I {} kubectl --kubeconfig /.../credentials/kubeconfig_tidb-cluster patch pv {} -p '{"spec":{"persistentVolumeReclaimPolicy":"Delete"}}'
     kubeconfig_file = ./credentials/kubeconfig_tidb-cluster
-    monitor_lb_ip = 35.227.134.146
-    monitor_port = 3000
-    region = us-west1
-    tidb_version = v3.0.1
     ```
 
+## 部署 TiDB 集群和监控
+
+1. 准备 TidbCluster 和 TidbMonitor CR 文件：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+	  cp manifests/{db,db-monitor}.yaml .
+    ```
+
+    参考 [API 文档](https://github.com/pingcap/tidb-operator/blob/master/docs/api-references/docs.html)和[集群配置文档](configure-cluster-using-tidbcluster.md)完成 CR 文件配置。
+
+    > **注意：**
+    >
+    > * 请使用 GKE 部署过程中配置的 `default_tidb_cluster_name` (默认为 `tidb-cluster`) 替换 `db.yaml` 和 `db-monitor.yaml` 文件中所有的 `CLUSTER_NAME`。
+    > * 请确保 GKE 部署过程中 PD、TiKV 或者 TiDB 节点的数量的值，与 `db.yaml` 中对应组件的 `replicas` 字段值一致。注意 Regional 集群下，实际创建的节点数为 pd_count/tikv_count/tidb_count 的 3 倍。
+    > * 请确保 `db-monitor.yaml` 中 `spec.initializer.version` 和 `db.yaml` 中 `spec.version` 一致，以保证监控显示正常。
+
+2. 创建 `Namespace`：
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    kubectl --kubeconfig credentials/kubeconfig_<gke_name> create namespace <namespace>
+    ```
+
+    > **注意：**
+    >
+    > `namespace` 是[命名空间](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/)，可以起一个方便记忆的名字，比如和 `default_cluster_name` 相同的名称。
+3. 部署 TiDB 集群：
+
+  {{< copyable "shell-regular" >}}
+
+  ```shell
+  kubectl --kubeconfig credentials/kubeconfig_<gke_name> create -f db.yaml -n <namespace>
+  kubectl --kubeconfig credentials/kubeconfig_<gke_name> create -f db-monitor.yaml -n <namespace>
+  ```
+
 ## 访问 TiDB 数据库
 
 `terraform apply` 运行完成后，可执行以下步骤来访问 TiDB 数据库。注意用[部署 TiDB 集群](#部署-tidb-集群)小节的输出信息替换 `<>` 部分的内容。
 
-1. 通过 `ssh` 远程连接到堡垒机。
+1. 获取 tidb Internal LoadBalancer IP 地址:
+
+	```shell
```

Comment:
Don't use hard tab to indent. Use 4 spaces. Please also replace other tabs in this document with spaces. Thanks!
```suggestion
    ```shell
```